File: qe-java-selenium/Activity18.java
package demos;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;
import java.time.Duration;

public class Activity18 {
    public static void main(String[] args) {
        
        WebDriver driver = new FirefoxDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));

        
        driver.get("https://training-support.net/webelements/alerts");

        System.out.println("Page title is: " + driver.getTitle());

        
        driver.findElement(By.id("simple")).click();

        
        wait.until(ExpectedConditions.alertIsPresent());

        
        Alert alert = driver.switchTo().alert();

        
        System.out.println("Alert text is: " + alert.getText());

        alert.accept();
        driver.quit();
    }
}

File: qe-selenium-python/Activity18.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()
wait = WebDriverWait(driver, 10)

driver.get("https://training-support.net/webelements/alerts")

print("Page title is:", driver.title)

driver.find_element(By.ID, "simple").click()

wait.until(EC.alert_is_present())

alert = driver.switch_to.alert

print("Alert text is:", alert.text)

alert.accept()

driver.quit()